src/main/java/api/InstrumentApi.java

```diff
@@ -0,0 +1,13 @@
+package api;
+
+import model.Instrument;
+import spark.Request;
+import spark.Response;
+
+import java.util.List;
+
+public class InstrumentApi {
```

Comment:
Czy my to będziemy rozszerzać w stylu czy dajemy możliwość dodawania instrumentów itd?